CHANGES

```diff
@@ -9,6 +9,28 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Deprecated low level ECDH and ECDSA functions. These include:
+         ECDH_compute_key, ECDSA_do_sign, ECDSA_do_sign_ex, ECDSA_do_verify,
+         ECDSA_sign_setup, ECDSA_sign, ECDSA_sign_ex, ECDSA_verify and
+         ECDSA_size.
```

Comment:
I suggest looking at previous examples for enumerations like this. I would intuitively add en empty line before the enumeration, and no extra indentation. I believe that's the style we have used previously.

Reply:
It's been variable previously.  I'll do something :)

Reply:
I reformatted this and the few after to the blank line before and after without indentation style.